apps/examples/src/app/basic-smart-dumb-example/basic-smart/basic-smart.component.ts

```diff
@@ -16,7 +16,7 @@ import { AsyncPipe } from '@angular/common';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class BasicSmartComponent {
-  private readonly service = inject(RandomBreweryService);
+  constructor(private readonly service: RandomBreweryService) {}
```

Comment:
There's no need to stop using the `inject` function and go back to constructor-based injection.  The tests should be fixable by just adding a `jest.mock('@angular/core');` at the top of them.